### PR TITLE
Interview guide

### PR DESCRIPTION
**Pull request by MatthijsMentink**
i did this because it is more than preparing, but does not fully help
you how to interview. so i thought, working on is a fine discription.

### CODE REVIEW

File: interview-guide/readme.md [hunk 1]
```diff
@@ -55,3 +61,23 @@ Don't be afraid of uncomfortable silences, sometimes you interviewee needs time
 
 ### Reboot an interview
 If an interview is going badly, you can try to reboot the interview by thanking the participant for their time, and engaging in a little small talk. After a while ask your interviewee to answer one last question on the record and slowly reengage the interview. Often people will be more open to your questions after they spoke to you as a person for a bit and it feels as if there is less depending on there answers.
+
+## What to do after the interview.
+
+It is good write down what you thought was the most important things that where discussed, and make notes of why you think these things are important. Do this directly after the interview when the memory is still fresh. This way you have some reference when you are compiling the interview.
```
Comment (thomasdebeus) [inline interview-guide/readme.md]: > It is good write down what you thought was the most important things that where discussed and make notes of why you think these things are important.

Sentence does not flow well.

Suggestion: _It is good to write down key points of the discussed subjects and make notes of why you think these takeaways are important._

> the memory is still fresh.

Suggestion: _your memory is still fresh._
To make it more personal?

File: interview-guide/readme.md [hunk 2]
```diff
@@ -55,3 +61,23 @@ Don't be afraid of uncomfortable silences, sometimes you interviewee needs time
...
+
+### Transcribing
+
+The next step is to transcribe everything that was said by the interviewee. There are are numerous [tools](https://transcribe.wreally.com/signup) online services that can do this for you.
```
Comment (thomasdebeus) [inline interview-guide/readme.md]: > There are are numerous [tools](https://transcribe.wreally.com/signup) online services that can do this for you.  

Do you mean: There are are numerous [tools](https://transcribe.wreally.com/signup) (online services) that can do this for you.

File: interview-guide/readme.md [hunk 3]
```diff
@@ -55,3 +61,23 @@ Don't be afraid of uncomfortable silences, sometimes you interviewee needs time
 
 ### Reboot an interview
 If an interview is going badly, you can try to reboot the interview by thanking the participant for their time, and engaging in a little small talk. After a while ask your interviewee to answer one last question on the record and slowly reengage the interview. Often people will be more open to your questions after they spoke to you as a person for a bit and it feels as if there is less depending on there answers.
+
+## What to do after the interview.
+
+It is good write down what you thought was the most important things that where discussed, and make notes of why you think these things are important. Do this directly after the interview when the memory is still fresh. This way you have some reference when you are compiling the interview.
+
+### Transcribing
```
Comment (thomasdebeus) [inline interview-guide/readme.md:71]: Maybe explain in short what transcribing is.

File: interview-guide/readme.md [hunk 4]
```diff
@@ -55,3 +61,23 @@ Don't be afraid of uncomfortable silences, sometimes you interviewee needs time
...
+
+<img src="https://media4.giphy.com/media/1MayKbFuSKE1O/200.gif" width="100%">
+
+### Improving the interview until it is published
+
+The next three things can be done in order to improve an interview. Most of the time these things are done simultaneously, since it is hard to do one thing without the other and still get a publishable end result.
+
+* Filtering the information on relevance.
+* Organizing the information in the most logical order.
+* Adding textual or visual context where needed.
```
Comment (thomasdebeus) [inline interview-guide/readme.md:86]: Should this be an unordered or ordered list?

Comment (MatthijsMentink) [inline interview-guide/readme.md:86]: Like i said in the texst, they are done symultaneously, and on of the points says organize the ifromation in the most logical order, so am i missing somehting? 💃

Comment (thomasdebeus) [inline interview-guide/readme.md:86]: Cool so unorderd as it is sorry missed it 👍

**Pull request merged**